diff --git a/node_modules/strapi-provider-email-sendinblue/lib/index.js b/node_modules/strapi-provider-email-sendinblue/lib/index.js
index 06edfc6..db85bbd 100644
--- a/node_modules/strapi-provider-email-sendinblue/lib/index.js
+++ b/node_modules/strapi-provider-email-sendinblue/lib/index.js
@@ -15,9 +15,10 @@ const defaultClient = SibApiV3Sdk.ApiClient.instance;
  */
 const getOptions = function (config, options) {
   options = typeof options == 'object' ? options : {};
-  options.from = config.sendinblue_default_from || options.from;
-  options.fromName = config.sendinblue_default_from_name || options.fromName;
-  options.replyTo = config.sendinblue_default_replyto || options.replyTo;
+  options.from = options.from || config.sendinblue_default_from;
+  options.fromName = options.fromName ||config.sendinblue_default_from_name;
+  options.replyTo =  options.replyTo || config.sendinblue_default_replyto;
+  options.cc =  options.cc || config.sendinblue_default_cc;
   options.text = options.text || options.html;
   options.html = options.html || options.text;
   return options;
@@ -39,8 +40,10 @@ const init = (config) => {
       return new Promise((resolve, reject) => {
         options = getOptions(config, options);
         sendSmtpEmail.sender = {email: options.from, name: options.fromName};
+        // sendSmtpEmail.sender = {email: options.from, name: options.fromName};
         sendSmtpEmail.replyTo = {email: options.replyTo};
         sendSmtpEmail.to = [{email: options.to}];
+        sendSmtpEmail.cc = [{email: options.cc}];
         sendSmtpEmail.subject = options.subject;
         sendSmtpEmail.htmlContent = options.html;
         sendSmtpEmail.textContent = options.text;
